README.md

```diff
@@ -16,3 +16,17 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+------------------------------------------------------------------------------
+
+I would start by running different sizes of array through the algorithm, so say that the I feed it an array of size 10 and it takes around 15 milliseconds to finish.  With this base line we can run a few more different sizes of array to find a trend.  Using a potential graph of $y=x+5$ So say that we then run an array of size 100 through it, and it takes around 105 milliseconds, and again on an array of size 1000 and this one takes around 1005 milliseconds, as we can see this way they show that it is supporting the researchers claim of $O(n)$ time.  But say that if we were doing the same method but had different results.  If we run an array of size 10 and still get 15 milliseconds as a start we can compare more arrays again, so if we do an array of size 100 but get 205 milliseconds, this change is noticeable but two points don't tell us much, so if we run an array of size 1000 into it and get around 3000 milliseconds we can tell that this is not increasing in a linear fashion, but in $O(nlogn)$ (times from $y=xlogx$) time because if it was linear the expected 3rd time on an array of 1000 would be around 2103 milliseconds with an equation I built using those two points of $y=2.11x-6.11$
```

Comment:
Where are you getting these times from?

Reply:
I was using just potential graphs of y = x + 5, y = xlogx + 5, and then a calculated y = 2.11x + 6.11.  The first linear equation might not be the actual supposed complexity equation of the fake algorithm but I'm using it to just show a linear increase using points along that line to show what a linear trend might look like, namely {(10,15), (100, 105), (1000, 1005)} and then from the xlogx graph the two points {(10, 15), (100, 205)} which I then used those two points to find the linear equation y = 2.11x + 6.11 to show that the third time value in a linear trend at 1000 would be significantly different to the same array size in a logarithmic trend.  So I got the values from graphs I just made up to get example trends on a coordinate plane where x = array size, and y = milliseconds.  Sorry if that's a lot